{"name": "Проверка ошибки при попытке авторизации пользователя c неверным паролем", "status": "passed", "steps": [{"name": "POST запрос на авторизацию пользователя", "status": "passed", "parameters": [{"name": "payload_for_login_user", "value": "{'email': 'rlminlgk@yandex.ru', 'password': 'jvruccg'}"}], "start": 1719134141354, "stop": 1719134142046}], "start": 1719134141354, "stop": 1719134142047, "uuid": "4ab49ea2-4f4d-4c0c-ab9f-4904c9a71c28", "historyId": "57298b213e07d772b396eff82bad7a26", "testCaseId": "57298b213e07d772b396eff82bad7a26", "fullName": "tests.test_2_login_user.TestLoginUser#test_login_user_with_invalid_password", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_2_login_user"}, {"name": "subSuite", "value": "TestLoginUser"}, {"name": "host", "value": "DESKTOP-5IEA5R0"}, {"name": "thread", "value": "14448-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_2_login_user"}]}